blog/community-update-wasmcloud-incubation-process/index.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: community-update-wasmcloud-incubation-process
+title: 'Community Update: wasmCloud Incubation Process'
+image: '/img/wasmcloud_large_social.png'
+date: 2024-05-28T09:00:00+05:00
+author: 'Taylor Thomas'
+author_profile: 'https://twitter.com/_oftaylor'
+description: "An update on wasmCloud's Incubation Process"
+draft: false
+---
+
+Hey wasmCloud Community!
+
+We wanted to give you an update on our journey toward [incubation status](https://github.com/cncf/toc/issues/1338) with the Cloud Native Computing Foundation (CNCF). As some of you might have noticed, the process has been taking a little longer than we initially anticipated. But don't worry–there’s a good reason for the delay. The CNCF Technical Oversight Committee (TOC) has been hard at work streamlining the entire incubation process and we're one of the first projects to go through this updated process. Although it has been a delay for us, it is going to be a win for the larger CNCF community now that they've adopted this more streamlined process.
```

Comment:
process is taking a little longer

Reply:
Don't worry, there's a good reason for the delay.

Reply:
Although it has been a delay - we can remove this as repetition. 

Start sentence: Adopting this more streamlined process will be a huge win for the broader CNCF community.